tools/src/format.rs

```diff
@@ -498,58 +497,55 @@ pub fn format_file_data(
                                class, revs, filespecs, blame_linenos);
             data
         } else {
-            "".to_owned()
+            // If we have no blame data, we want the div here taking up space, but we don't want it
+            // to both screen readers.
```

Comment:
comment here seems to be missing a word or something

Reply:
Yes, I meant s/both/bother/ here.